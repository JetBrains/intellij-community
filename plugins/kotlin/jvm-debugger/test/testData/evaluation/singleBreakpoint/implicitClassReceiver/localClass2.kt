// ATTACH_LIBRARY: simpleOpenClass

import forTests.*

fun main() {
    class Inner : SuperClass() {
        override fun bar(): Int {
            // EXPRESSION: this
            // RESULT: instance of LocalClass2Kt$main$Inner(id=ID): LLocalClass2Kt$main$Inner;
            //Breakpoint!
            return 2
        }
    }

    Inner().bar()
}


// PRINT_FRAME