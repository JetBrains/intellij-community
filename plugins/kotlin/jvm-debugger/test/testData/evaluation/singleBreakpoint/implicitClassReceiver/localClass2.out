LineBreakpoint created at localClass2.kt:11
Run Java
Connected to the target VM
localClass2.kt:11
Compile bytecode for this
KotlinStackFrame (localClass2.kt:11)
    JavaValue[this] this: LocalClass2Kt$main$Inner@uniqueID = LocalClass2Kt$main$Inner@hashCode
        DummyMessageValueNode = Class has no properties
Disconnected from the target VM

Process finished with exit code 0
